src/main/js/main_html5.js

```diff
@@ -859,24 +861,51 @@ require("../../../html5-common/js/utils/environment.js");
      * For multi audio we can get a list of available audio tracks
      * @public
      * method OoyalaVideoWrapper#getAvailableAudio
-     * @returns {(boolean|Array)} false - if an array with tracks is not available or Array - an array of all available audio tracks.
+     * @returns {Array} - an array of all available audio tracks.
      */
     this.getAvailableAudio = function() {
-      var audioTrack = _video.audioTracks;
-      if (audioTrack !== undefined && audioTrack.length) {
+      var audioTracks = _video.audioTracks;
+      if (audioTracks !== undefined && audioTracks.length) {
         var audioTrackList = [];
-        for (var i = 0; i < audioTrack.length; i++) {
+        for (var index = 0; index < audioTracks.length; index++) {
           var element = {
-            id: audioTrack[i].id,
-            kind: audioTrack[i].kind,
-            label: audioTrack[i].label,
-            lang: audioTrack[i].language,
-            enabled: audioTrack[i].enabled
+            id: audioTracks[index].id,
+            kind: audioTracks[index].kind,
+            label: audioTracks[index].label,
+            lang: audioTracks[index].language,
+            enabled: audioTracks[index].enabled
           };
+          if (audioTracks[index].enabled) {
+            currentAudioId = audioTracks[index].id;
+          }
           audioTrackList.push(element);
         }
         return audioTrackList;
       }
+      return [];
```

Comment:
Why is this necessary? As I see, you are using an empty array and filling it with objects from the audioTracks. If the audioTracks ==[], then you can return the audioTrackList. It will also be empty.